src/expr/src/scalar/func.rs

```diff
@@ -5736,7 +5760,9 @@ pub enum VariadicFunc {
     ArrayToString {
         elem_type: ScalarType,
     },
-    ArrayIndex,
+    ArrayIndex {
+        offset: usize,
```

Comment:
Does the offset indicate whether the array is 0 indexed or 1 indexed? If so, if you agree, it might benefit from a small doc comment indicating that.

Reply:
Sure thing